pkg/v1/codegen/doc.go

```diff
@@ -1,5 +0,0 @@
-// Copyright 2021 VMware, Inc. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-// Package codegen provides code generators
-package codegen
```

Comment:
I don't know what these files are for to be honest, but why are we removing this one?

Reply:
i moved the code from `pkg/v1/codegen/` to `cmd/cli/plugin-admin/code-gen` and we don't a doc.go for main package, so deleted that file